dftparse/vasp/eigenval_parser.py

```diff
@@ -0,0 +1,71 @@
+from ..core import BlockParser
+
+
+def _is_kpoint(line):
+    """Is this line the start of a new k-point block"""
+    # Try to parse the k-point; false otherwise
+    toks = line.split()
+    try:
+        return len(toks) == 4 and all([float(x) < 1.0 for x in toks])
```

Comment:
Is vasp good enough about precision for this to be safe? What about `float(x) > -1e-8 and float(x) < 1+1e-8` or something to be safe?